itest/assertions.go

```diff
@@ -716,7 +716,6 @@ func AssertNonInteractiveRecvComplete(t *testing.T,
 func AssertAddr(t *testing.T, expected *taprpc.Asset, actual *taprpc.Addr) {
 	require.Equal(t, expected.AssetGenesis.AssetId, actual.AssetId)
 	require.Equal(t, expected.AssetType, actual.AssetType)
-	require.Equal(t, expected.Version, actual.AssetVersion)
```

Comment:
Shouldn't we keep this assertion?

Reply:
So this ended up being misplaced: originally intent was to ensure that when you made an addr, it matched the asset created. However, in the tests, we want to ensure we can make an addr a v0 asset for a v1 asset (since they're more or less isomorphic) rn. Otherwise, we would never be able to make a v0 addr for a v1 asset with the itests the way tehy're set up rn.

Reply:
Turns out just forgot to _also_ set the version in the `AssetGenesis` struct, we have it duplicated across the main asset and this genesis version field as well.